app/templates/_pom.xml

```diff
@@ -694,6 +694,28 @@
             <plugin>
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>default-test</id>
+                        <configuration>
+                            <excludes>
+                                <exclude>**/Abstract*.java</exclude>
+                                <exclude>**/*Cucumber*.java</exclude>
+                            </excludes>
+                        </configuration>
+                    </execution>
```

Comment:
it works for all current config, but shouldn't we add this new code only for cucumber ?